### PR TITLE
added login & signup pages

### PR DESCRIPTION
**Pull request by tusharkandpal**
- login & signup html consists of navbar, main, section, & form container
- styles are almost common for both

NOTE: will create color variables in next PR.

### CODE REVIEW

File: components/login-page/login.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/* ------------------------|| IMPORTS -----------------------*/
+@import url("../../utilities/main.css");
+
+/* ------------------------- || LOGIN MAIN ----------------------- */
+.login-main {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 85vh;
+  background-color: #f7f7f7;
```
Comment (ozairkhan2810) [inline components/login-page/login.css:10]: use color variable

**Review by sushil-mod — APPROVED** [review]
good code ,login-main and signup-main class have same properties use one class if possible, rest is good ,you can merge

**Review by ozairkhan2810 — APPROVED** [review]
use color variables if you can, rest looks good

**Comment by tusharkandpal** [comment]
> use color variables if you can, rest looks good

i mentioned in PR description.. will take in next one

**Comment by tusharkandpal** [comment]
> good code ,login-main and signup-main class have same properties use one class if possible, rest is good ,you can merge

i was thinking to use single css file for both but didn't eventually

**Pull request merged**